src/components/StateBar/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { css } from '@emotion/react'
 import PropTypes from 'prop-types'
 import React, { ReactNode, useMemo } from 'react'
+import { colors } from '../../theme'
```

Comment:
We try to avoid importing directly from the theme file, could you use the injected Theme in emotion instead ?